library/std/src/sys/unix/process/process_common.rs

```diff
@@ -209,6 +293,9 @@ impl Command {
     pub fn pgroup(&mut self, pgroup: pid_t) {
         self.pgroup = Some(pgroup);
     }
+    pub fn signal_mask(&mut self, mask: SignalSet) {
```

Comment:
This should accept an `Into<Option<SignalSet>>`, to allow passing `None` as well.